physics/dynamic_frame_body.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "physics/dynamic_frame.hpp"
+
+namespace principia {
+namespace physics {
+template <typename FromFrame, typename ToFrame>
```

Comment:
No space before `<`.  I think that's what we generally do, and that's what you did in the declaration.